service-base/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-service-base/native-image.properties

```diff
@@ -14,4 +14,5 @@ Args = -H:ResourceConfigurationResources=${.}/resources-config.json \
   --initialize-at-run-time=io.netty.internal.tcnative.SSL \
   -Dvertx.json.base64=legacy \
   -Djdk.tls.ephemeralDHKeySize=matched \
-  -Djdk.tls.rejectClientInitiatedRenegotiation=true
+  -Djdk.tls.rejectClientInitiatedRenegotiation=true \
+  -H:+RunReachabilityHandlersConcurrently
```

Comment:
this needs to (only) be put into the concrete service implementation modules so that the native-build command applies it *last*. If put into one of the dependencies, it is uncertain whether it will be applied before *gax-rpc* sets it to the opposite value ...

Reply:
Removed.